forge-gui/res/cardsfolder/upcoming/bladewing_deathless_tyrant.txt

```diff
@@ -0,0 +1,11 @@
+Name:Bladewing, Deathless Tyrant
+ManaCost:5 B R
+Types:Legendary Creature Dragon Skeleton
+PT:6/6
+K:Flying
+K:Haste
+T:Mode$ DamageDone | ValidSource$ Card.Self | ValidTarget$ Player,Planeswalker | CombatDamage$ True | Execute$ TrigToken | TriggerZones$ Battlefield | TriggerDescription$ Whenever CARDNAME deals combat damage to a player or planeswalker, for each creature card in your graveyard, create a 2/2 black Zombie Knight creature token with menace.
+SVar:TrigToken:DB$ Token | TokenScript$ b_2_2_zombie_knight_menace | TokenAmount$ X | TokenOwner$ You
+SVar:X:Count$TypeInYourYard.Creature
+DeckHas:Ability$Graveyard & Type$Zombie|Knight
```

Comment:
DeckHas:Ability$Token & Type$Zombie|Knight

Maybe:
DeckHints:Ability$Graveyard

Reply:
Reworked DeckHas and DeckHints flags as recommended